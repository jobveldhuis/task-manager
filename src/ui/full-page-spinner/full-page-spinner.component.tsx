import { ActivityIndicator, StyleSheet, View } from "react-native";
import { COLORS } from "../../util/colors.const";

export function FullPageSpinner() {
  return (
    <View style={styles.container}>
      <ActivityIndicator size="large" color={COLORS.secondary} />
    </View>
  );
}

const styles = StyleSheet.create({
  container: {
    display: "flex",
    minWidth: "100%",
    minHeight: "100%",
    paddingTop: 16,
    justifyContent: "flex-start",
    alignItems: "center",
  },
});
