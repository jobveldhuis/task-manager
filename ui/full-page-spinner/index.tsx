export { FullPageSpinner } from "./full-page-spinner.component";
