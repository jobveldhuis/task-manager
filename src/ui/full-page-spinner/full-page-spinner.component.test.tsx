import { render } from "@testing-library/react-native";
import { FullPageSpinner } from "@/ui/full-page-spinner/full-page-spinner.component";

describe("FullPageSpinner", () => {
  it("should match snapshot", () => {
    expect(render(<FullPageSpinner />).toJSON()).toMatchSnapshot();
  });
});
