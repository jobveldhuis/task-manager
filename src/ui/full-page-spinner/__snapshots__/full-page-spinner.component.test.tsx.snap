// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullPageSpinner should match snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "display": "flex",
      "justifyContent": "flex-start",
      "minHeight": "100%",
      "minWidth": "100%",
      "paddingTop": 16,
    }
  }
>
  <ActivityIndicator
    color="#000"
    size="large"
  />
</View>
`;
